### ISSUE TITLE
GraphQL should also support request with POST method and Content-Type: application/json 

### ISSUE THREAD

**Issue by ttu** [ttu/dotnet-fake-json-server#13]
Request handling is in [FakeServer/GraphQL/GraphQLMiddleware.cs](https://github.com/ttu/dotnet-fake-json-server/blob/master/FakeServer/GraphQL/GraphQLMiddleware.cs).

#### Now middleware supports:

If the "application/graphql" Content-Type header is present, treat the HTTP POST body contents as the GraphQL query string.

#### Should also support this:

If the "application/json " Content-Type header is present, treat the HTTP POST body contents as a JSON with parameter query having the GraphQL query string.

```
Method: POST 
Content-Type: application/json 
Body: { "query": "{users{name}}" }
```

**Comment by antoniaelek** [comment]
Gonna give this one a go if that's alright. Also #12.

**Comment by ttu** [comment]
Implemented with #24
